listenbrainz/webserver/templates/art/svg-templates/yim-2022-playlists.svg

```diff
@@ -0,0 +1,37 @@
+<svg
+    width="452"
+    height="521"
+    viewBox="0 0 452 521"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+    <style>
+        .album-art {
+            preserveAspectRatio: "xMidYMid slice";
+            height: 135px;
+            width: 135px;
+        }
+    </style>
+    <g>
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="30" y="75" xlink:href="{{ image_urls[0] }}" />
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="30" y="210" xlink:href="{{ image_urls[1] }}" />
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="30" y="345" xlink:href="{{ image_urls[2] }}" />
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="165" y="75" xlink:href="{{ image_urls[3] }}" />
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="165" y="210" xlink:href="{{ image_urls[4] }}" />
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="165" y="345" xlink:href="{{ image_urls[5] }}" />
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="300" y="75" xlink:href="{{ image_urls[6] }}" />
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="300" y="210" xlink:href="{{ image_urls[7] }}" />
+        <image class="album-art" preserveAspectRatio="xMidYMid slice" x="300" y="345" xlink:href="{{ image_urls[8] }}" />
+        <rect x="0" y="0" width="452" height="521" fill="url(#pattern2)"/>
+    </g>
+    <defs>
+        <pattern id="pattern0" patternContentUnits="objectBoundingBox" width="1" height="1">
+            <use xlink:href="#bg" transform="translate(-0.067113 -0.468785) scale(0.00127425 0.00195211)"/>
+        </pattern>
+        <pattern id="pattern2" patternContentUnits="objectBoundingBox" width="1" height="1">
+            <use xlink:href="#flames" transform="translate(-0.0891089 -0.0386266) scale(0.000479208 0.000415451)"/>
+        </pattern>
+        <image id="bg" width="1000" height="1000" xlink:href="{{ bg_image_url }}" />
```

Comment:
The background image is unused in this one, isn't it?
If so it should be remove, along with the pattern0 above

Reply:
Done.